src/main/java/edu/kis/powp/jobs2d/features/DrawerFeature.java

```diff
@@ -6,29 +6,29 @@
 
 public class DrawerFeature {
 
-	private static DrawPanelController drawerController;
+    private static DrawPanelController drawerController;
 
```

Comment:
Unnecessary changes in code formatting. It makes tracking actual changes more difficult.